src/pages/index.js

```diff
@@ -181,41 +34,241 @@ const IndexPage = () => (
 
       <SprkStackItem>
         <div className="docs-o-Grid docs-o-Grid--home-page docs-o-Grid--large">
-          <SprkCard
-            additionalClasses="docs-c-Card"
-            variant="teaser"
-            teaserConfig={teaserDesigners}
-          />
-
-          <SprkCard
-            additionalClasses="docs-c-Card"
-            variant="teaser"
-            teaserConfig={teaserDevelopers}
-          />
-
-          <SprkCard
-            additionalClasses="docs-c-Card"
-            variant="teaser"
-            teaserConfig={teaserComponents}
-          />
-
-          <SprkCard
-            additionalClasses="docs-c-Card"
-            variant="teaser"
-            teaserConfig={teaserFoundations}
-          />
-
-          <SprkCard
-            additionalClasses="docs-c-Card"
-            variant="teaser"
-            teaserConfig={teaserUtils}
-          />
-
-          <SprkCard
```

Comment:
Why did you remove the SprkCard?

Reply:
This has been added back and new issues created to update SprkCard